src/components/Sum.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {a + b}
```

Comment:
[Checklist](https://github.com/mate-academy/react_sum/blob/master/checklist.md) 3rd item., use string interpolation for text